docs/community/3.14-announcement.md

```diff
@@ -0,0 +1,62 @@
+<style>
+.promo li a {
+    float: left;
+    width: 130px;
+    height: 20px;
+    text-align: center;
+    margin: 10px 30px;
+    padding: 150px 0 0 0;
+    background-position: 0 50%;
+    background-size: 130px auto;
+    background-repeat: no-repeat;
+    font-size: 120%;
+    color: black;
+}
+.promo li {
+    list-style: none;
+}
+</style>
+
+# Django REST framework 3.14
+
+## Django 4.1 support
+
+The latest release now fully supports Django 4.1.
+
+Our requirements are now:
+
+* Python 3.6+
```

Comment:
imho this should be 3.7+ but we need https://github.com/encode/django-rest-framework/pull/8518 to be merged
```suggestion
* Python 3.7+
```

Reply:
I'd be happy merging that but I'll defer to @tomchristie.